Sources/StatKit/Descriptive Statistics/Central Tendency/MeanType.swift

```diff
@@ -25,7 +25,7 @@ extension MeanType {
                                                              in collection: U) -> Double {
     switch self {
       case .arithmetic:
-        return collection.sum(over: variable).realValue / Double(collection.count)
+        return collection.average(over: variable)
```

Comment:
My suggestion is to inline the calculation part that resided in the Summation file, so the code that computes the mean is all in the same place.

```suggestion
        return zip(collection, 1...collection.count)
          .reduce(into: 0.0) { result, zip in
            let (element, count) = zip
            result += (element[keyPath: variable].realValue - result) / count.realValue
      }
```

Reply:
I was trying to use `enumerate()` to replace `zip()`, but it looks like a mess. 
```Swift
collection.enumerated().reduce(into: 0.0) {
            (result: inout Double, pair: EnumeratedSequence<U>.Element)  in
            let (index, element) = pair
            let count = index + 1  // terrible magic number `+1`
            result += (element[keyPath: variable].realValue - result) / count.realValue
        }
```
I think zip is a great choice!